src/transformers/modeling_t5.py

```diff
@@ -358,7 +358,10 @@ def unshape(x):
         else:
             present_key_value_state = (None,)
 
-        scores = torch.einsum("bnqd,bnkd->bnqk", q, k)  # (bs, n_heads, qlen, klen)
+        # (bs, n_heads, qlen, klen)
+        scores = q @ k.transpose(
```

Comment:
Can you replace the call to `@` operator to the actual `torch.matmul` operator? We tend to not use this notation in the repo 👍

Reply:
Sure thing! I just adjusted it to a torch.matmul